src/components/news-form/news-form.js

```diff
@@ -158,6 +159,16 @@ const NewsForm = ({ article, id }) => {
           </TabPanel>
         ))}
       </form>
+
+      <div className={classes.controlsBlock}>
+        <StandardButton
+          id='back'
```

Comment:
Use the same label as data-cy attribute has

Reply:
done